crates/re_dev_tools/src/build_search_index/util.rs

```diff
@@ -131,3 +133,19 @@ impl CheckStatus for std::process::Output {
         self.status.check()
     }
 }
+
+pub trait ProgressBarExt {
+    fn set(&self, message: impl Into<Cow<'static, str>>, is_tty: bool);
+}
+
+impl ProgressBarExt for ProgressBar {
+    fn set(&self, message: impl Into<Cow<'static, str>>, is_tty: bool) {
+        if is_tty {
```

Comment:
I expected `indicatif` to print any message changes when stdout is not a TTY, but it doesn't, it just prints nothing at all, which made it very hard to diagnose where the build failed.

Reply:
That should be a comment in the code, not here!